oap-server/server-library/library-elasticsearch-client/src/main/java/org/apache/skywalking/library/elasticsearch/ElasticSearchVersion.java

```diff
@@ -43,7 +43,7 @@ private ElasticSearchVersion(final String distribution, final int major, final i
 
         if (distribution.equalsIgnoreCase("OpenSearch")) {
             requestFactory = new V78RequestFactory(this);
-            codec = V7Codec.INSTANCE;
+            codec = V78Codec.INSTANCE;
```

Comment:
Forgot to update to `V78Codec` after splitting the de/ser for `[7.0, 7.8)` `[7.8, 8.0)`